extensions/sql-migration/package.json

```diff
@@ -143,6 +143,18 @@
         }
       ]
     },
+    "configuration": [
+      {
+        "title": "%migration-dashboard-title%",
+        "properties": {
+          "azureRecommendation.dataCollectionPath": {
+            "type": "string",
+            "default": "%userprofile%\\AppData\\Roaming\\azuredatastudio\\PerfData",
```

Comment:
This isn't going to work on Linux/Mac - do you only support Windows with this extension?

Unfortunately there isn't support for platform-specific defaults that I'm aware of, but you could at least have the default be blank and then if the config is empty have your code default to an appropriate path on each platform instead. (and include this note in the description)

Reply:
@Charles-Gagnon, in default we are just showing a string value as default to let user know what the current default path is.  So "%userprofile%" is shown as string rather than actual value of it.

"This isn't going to work on Linux/Mac - do you only support Windows with this extension?" - Do you mean the default path would be different in Linux/Mac than this?

Reply:
It should be, because otherwise this default value will not work for Linux/Mac users (since it's not a valid path).

Unfortunately there isn't a way to do platform-specific defaults. So what I've typically seen done is just to have the default be blank and then in your code where you read this value have it default to the correct platform-specific default value at that time instead - with the description of the config informing the user what the default will be for the current platforms.

Reply:
Thanks @Charles-Gagnon. I understood the issue. Hence, I made the default to be blank.